Library/Formula/cassandra.rb

```diff
@@ -28,23 +67,21 @@ def install
 
     rm Dir["bin/*.bat", "bin/*.ps1"]
 
+    # This breaks on `brew uninstall cassandra && brew install cassandra`
+    # https://github.com/Homebrew/homebrew/pull/38309
     (etc+"cassandra").install Dir["conf/*"]
-    prefix.install Dir["*.txt", "{bin,interface,javadoc,pylib,lib/licenses}"]
-    prefix.install Dir["lib/*.jar"]
 
-    share.install [bin+'cassandra.in.sh', bin+'stop-server']
-    inreplace Dir["#{bin}/cassandra*", "#{bin}/debug-cql", "#{bin}/nodetool", "#{bin}/sstable*"],
-              /`dirname "?\$0"?`\/cassandra.in.sh/,
-              "#{share}/cassandra.in.sh"
-  end
+    libexec.install Dir["*.txt", "{bin,interface,javadoc,pylib,lib/licenses}"]
+    libexec.install Dir["lib/*.jar"]
+    bin.write_exec_script Dir["#{libexec}/bin/*"]
 
-  def caveats; <<-EOS.undent
-    If you plan to use the CQL shell (cqlsh), you will need the Python CQL library
-    installed. Since Homebrew prefers using pip for Python packages, you can
-    install that using:
+    rm bin/"cqlsh" # Remove existing exec script
+    (bin/"cqlsh").write_env_script libexec/"bin/cqlsh", :PYTHONPATH => pypath
 
-      pip install cql
-    EOS
+    share.install [libexec+"bin/cassandra.in.sh", libexec+"bin/stop-server"]
```

Comment:
This leaves a broken stop-server stub in bin; do this before writing the exec scripts.